sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorLogExporter.cs

```diff
@@ -29,7 +29,7 @@ internal AzureMonitorLogExporter(ITransmitter transmitter)
             _instrumentationKey = transmitter.InstrumentationKey;
             _resourceParser = new ResourceParser();
 
-            if (transmitter is AzureMonitorTransmitter _azureMonitorTransmitter && _azureMonitorTransmitter._storage != null)
+            if (transmitter is AzureMonitorTransmitter _azureMonitorTransmitter && _azureMonitorTransmitter._fileBlobProvider != null)
```

Comment:
nit: why does `_azureMonitorTransmitter` have the underscore?
I would expect that for private/internal fields, but not a local variable.